.github/workflows/mirror-dev.yml

```diff
@@ -0,0 +1,29 @@
+name: Mirror dev to GitLab
+
+on:
+  push:
+    branches:
+      - dev
+
+jobs:
+  mirror:
+    if: github.repository == 'accessibility-exchange/platform'
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Git
+        run: |
+          git config --global user.name "${{ vars.GIT_USER }}"
+          git config --global user.email "${{ vars.GIT_EMAIL }}"
+          git remote add gitlab ${{ vars.GITLAB_URL }}
+
+      - name: Push to GitLab
+        run: |
+          mkdir -p ~/.ssh
+          echo "${{ secrets.GITLAB_SSH_KEY }}" > ~/.ssh/id_rsa
+          chmod 600 ~/.ssh/id_rsa
+          ssh-keyscan gitlab.com >> ~/.ssh/known_hosts
```

Comment:
You've used `webfactory/ssh-agent@v0.7.0` in the other workflows, did you mean to use that here too?

Reply:
Thanks for catching that. I should have reviewed after merging stashed changes.